### PR TITLE
AHTI-419 | Reset database to a DB dump

### PR DESCRIPTION
**Pull request by charn**
## Description :sparkles:
This code should enable the production environment to restore/reset the DB to a state of the provided database dump. Magic wormhole will be used for delivering the DB dump to the image from a developer machine.

### Closes :no_good_woman:
**[AHTI-419](https://helsinkisolutionoffice.atlassian.net/browse/AHTI-419):** BE: Migrate data from staging (QA) to production

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ services:
         env_file:
             - docker-compose.env.yaml
         environment:
-            DATABASE_URL: postgis://ahti:ahti@ahti-db/ahti
+            DATABASE_URL: postgres://ahti:ahti@ahti-db/ahti
```
Comment (igordavydsson) [inline docker-compose.yml:23]: ahti not using gis anymore?

Comment (charn) [reply docker-compose.yml:23]: It is.. it's enforced in `settings.py`. `psql` command understands only db addresses that start with `postgres://`. AFAIK the DATABASE_URL configured ahti in the CI/CD pipeline both begin with `postgres`.

Comment (charn) [reply docker-compose.yml:23]: Basically I wanted to avoid adding a second setting for a database url with this particular change.

**Comment by charn** [comment]
Indeed, a lot of depencies are added, but once the production is up and running smoothly it should be fine to get rid of the code in this PR. :sweat_smile:

File: docker-compose.yml (hunk 1 above)
Comment (igordavydsson) [reply docker-compose.yml:23]: ah, okay 🙂

**Pull request merged**